### ISSUE TITLE
Plugin not working

### ISSUE THREAD

**Issue by mkoepl** [robertlemke/RobertLemke.Plugin.Blog#20]
Hi!
When creating a blogpost-overview, the following Exception is _thrown:

Exception in line 511 of /data/www/releases/current/Data/Temporary/Development/Cache/Code/Flow_Object_Classes/Neos_ContentRepository_ViewHelpers_Widget_Controller_PaginateController.php: 

177 Neos\ContentRepository\ViewHelpers\Widget\Controller\PaginateController_Original::indexAction(1)
176 Neos\ContentRepository\ViewHelpers\Widget\Controller\PaginateController::indexAction(1)
175 call_user_func_array(array|2|, array|1|)
174 Neos\ContentRepository\ViewHelpers\Widget\Controller\PaginateController::Flow_Aop_Proxy_invokeJoinPoint(Neos\Flow\Aop\JoinPoint)

For some reason, also the category page throws an exception:

The fluid file "resource://RobertLemke.Plugin.Blog/Private/Templates/NodeTypes/Category.html" was not found.

**Comment by breadlesscode** [comment]
Same here.

Neos Version: 3.3.3
Fow Version: 4.3.2

**Comment by kdambekalns** [comment]
Yes, that template does not exist… But the categories are not even output in the stock template. Does this happen with the plugin out-of-the-box?
